tools/arm64-build-win-vind.bat

```diff
@@ -0,0 +1,8 @@
+@REM  from https://github.com/pit-ray/win-vind/issues/191#issuecomment-1671656805
+
+call "c:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools\VsDevCmd.bat"
```

Comment:
Since the environment is automatically selected within cmake, there is no need to call `VsDevCmd.bat` here.

Reply:
`cmake` isn't in my path until I make that call. As noted above, I'm using VSCode.

Should I try this from VS-Studio (community version)? Will this call cause problems?

Reply:
When you call `VsDevCmd.cmd`, the paths of installed extensions into the Visual Studio are loaded.
We assume that the cmake is installed independently from Visual Studio, so you do not need to call the VsDevCmd.bat and need to install cmake independently.

You can install cmake in various ways, such as winget, Chocolatey, installer, and so on.
Please see also https://github.com/pit-ray/win-vind/blob/master/CONTRIBUTING.md#softwares .

Reply:
Okay - this is getting a bit complex to just try out the app. 😭 Let me spend some time on the x86 machine to see if `win-vind` will work for me, before I try to turn my machine into a dev box. 

That'll take a while (probably a month with everything else going on). Thanks for your help.

Reply:
I see. Please feel free to message me.